### PR TITLE
Created deu.ini

### PR DESCRIPTION
**Pull request by xLPMG**
Here´s the german translation! Everything should be right, since it´s my native language. However, if there happen to be any complaints about wrong translations, just message me and I´ll take care of it and fix them as soon as possible!
I hope i could help you with this!
Sincerely, LPMG.

### CODE REVIEW

**Comment by xLPMG** [comment]
!Note: Name could be changed to "ger.ini" because of the english name "German". Currently the german abbreviation for "Deutsch" is being used.

**Comment by Johnmacrocraft** [comment]
The filename deu.ini is ok. Also, can you check my checklist from translations submission issue template? It seems like you didn't use it for this PR. As a translator I highly enforce people to follow grammar, spelling etc.

File: resources/lang/deu.ini
Comment (Johnmacrocraft) [inline resources/lang/deu.ini]: The quotation mark isn't needed for this since your translation doesn't use semicolon in it. Also, the original string doesn't have period at the end. Can you remove it from your translation? Thanks!

**Review by Johnmacrocraft — DISMISSED** [review]
I have one change request for the translation.
Edit: one more

File: resources/lang/deu.ini
Comment (Johnmacrocraft) [inline resources/lang/deu.ini]: I don't speak German so I'm not sure, but is this following the translation of commands.generic.permission from Minecraft? For your better understanding, like mentioned from the above, this string is used with other permission strings like main.perm.generic, gui.buyinfo.perm.buy etc.

Original English string (from MCJE): You do not have permission to use this command
German translation (from MCJE): Du hast nicht die benötigte Berechtigung, diesen Befehl auszuführen

If it is following the style of translation from Minecraft then it's okay, but if it isn't, could you change your translation to follow it? Thanks!

**Review by Johnmacrocraft — COMMENTED** [review]
Please check the comment

File: resources/lang/deu.ini
Comment (xLPMG) [reply resources/lang/deu.ini]: Both mine and the official MCJE translation are perfectly fine, however I´d consider the official one slightly more formal. I agree to use that instead!

**Comment by xLPMG** [comment]
I hope everything is fixed now! In line 12 the word "member" is used, is it referring to the singular or plural form? 
Edit: Or both singular and plural?

File: resources/lang/deu.ini
Comment (xLPMG) [reply resources/lang/deu.ini]: I just left out the german word for "command" and "to use" as in "You don't have permission to {%0}" instead of "You do not have permission to use this command"

**Comment by Johnmacrocraft** [comment]
In line 12 string, the list of members of the newspaper is returned to its first parameter. It can be either only one player or more than it, so both.
Example:
Member: TestUser
Another example:
Member: TestUser, TestUser2, DebugUser123

**Comment by Johnmacrocraft** [comment]
For some reason some of the new lines are removed from the file, but it's not a big issue, so it doesn't matter. I'll add them after this gets merged!

**Comment by xLPMG** [comment]
The translation for member is right then. You could merge this now if there arent any more mistakes :p

**Comment by xLPMG** [comment]
i didnt mean to close!! it was by accident, sorry

**Comment by Johnmacrocraft** [comment]
Your translation is added to the plugin! Thank you for your translation submission. The translation will be included from the next release. Enjoy!

**Pull request merged**
